ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/FilterResourceDropdown.tsx

```diff
@@ -1,44 +1,54 @@
-import React, { useState } from 'react';
+import React from 'react';
 import { Select, SelectOption } from '@patternfly/react-core';
 
-import { SearchFilter } from 'types/search';
+import useSelectToggle from 'hooks/patternfly/useSelectToggle';
 
-export type Resource = 'CVE' | 'Image' | 'Deployment' | 'Namespace' | 'Cluster';
+export type Resource = 'IMAGE_CVE' | 'IMAGE' | 'DEPLOYMENT' | 'NAMESPACE' | 'CLUSTER';
```

Comment:
so `IMAGE_CVE` is the category we have to use for the autocomplete query, which is why i changed it here. i can look into changing it only for the autocomplete input instead of across the logic here though